core/io/resource_format_binary.cpp

```diff
@@ -891,10 +896,24 @@ void ResourceLoaderBinary::open(FileAccess *p_f) {
 	if (flags & ResourceFormatSaverBinaryInstance::FORMAT_FLAG_NAMED_SCENE_IDS) {
 		using_named_scene_ids = true;
 	}
-	for (int i = 0; i < 13; i++) {
+	if (flags & ResourceFormatSaverBinaryInstance::FORMAT_FLAG_UIDS) {
+		using_uids = true;
+	}
+
+	if (using_uids) {
+		uid = f->get_64();
+	} else {
+		uid = ResourceUID::INVALID_ID;
+	}
+
+	for (int i = 0; i < 5; i++) {
```

Comment:
Wasn't this supposed to be `11`, rather than `5`? We had 13 32-bit reserved fields before, then we added a new 64-bit field, so we would be left with 11 fields. This change breaks backwards compatibility with previous binary resource formats.